src/ansys_sphinx_theme/extension/linkcode.py

```diff
@@ -197,8 +197,6 @@ def link_code(app: Sphinx, doctree: Node):
     The `link_code_library` configuration option can be used to specify a custom library
     for link resolution.
```

Comment:
```suggestion
    You can use the ``link_code_library`` configuration option to specify a custom library
    for link resolution.
```
I'd like to make some edits to the preceding lines but I can't do so in this PR. There are additional places where single backticks are used instead of double backticks. Single backticks render as italics in the doc (rather than as a code entity).